<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="gbShema">
      <property name="title">
       <string>Структура информационной сети</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="separator"/>
    <addaction name="action_save_image"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Мнемосхема</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="separator"/>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
    <addaction name="separator"/>
    <addaction name="action_12"/>
    <addaction name="action_13"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionNew"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_add_comp"/>
   <addaction name="action_add_router"/>
   <addaction name="action_add_radio"/>
   <addaction name="action_add_linc"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShowInfo"/>
   <addaction name="actionConfig"/>
   <addaction name="separator"/>
   <addaction name="actionRouteInform"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>72</width>
     <height>72</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_route1"/>
   <addaction name="action_route2"/>
   <addaction name="action_route3"/>
  </widget>
  <action name="action_new">
   <property name="text">
    <string>Новая мнемосхема</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Открыть мнемосхему</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Сохранить мнемосхему</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>Сохранить мнемосхему как ...</string>
   </property>
  </action>
  <action name="action_save_image">
   <property name="text">
    <string>Сохранить изображение</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_add_comp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/workstation_256.png</normaloff>:/img/img/workstation_256.png</iconset>
   </property>
   <property name="text">
    <string>computer</string>
   </property>
   <property name="toolTip">
    <string>Добавить компьютер</string>
   </property>
  </action>
  <action name="action_add_router">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/router-hi.png</normaloff>:/img/img/router-hi.png</iconset>
   </property>
   <property name="text">
    <string>router</string>
   </property>
   <property name="toolTip">
    <string>Добавить маршрутизатор</string>
   </property>
  </action>
  <action name="action_add_linc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/connect_256.png</normaloff>
     <normalon>:/img/img/disconnect_256.png</normalon>:/img/img/connect_256.png</iconset>
   </property>
   <property name="text">
    <string>ethernet</string>
   </property>
   <property name="toolTip">
    <string>Добавить узел</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/folder_256.png</normaloff>:/img/img/folder_256.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
   <property name="toolTip">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/folder_add_256.png</normaloff>:/img/img/folder_add_256.png</iconset>
   </property>
   <property name="text">
    <string>new</string>
   </property>
   <property name="toolTip">
    <string>Новая мнемосхема</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/folder_save_256.png</normaloff>:/img/img/folder_save_256.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="toolTip">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionShowInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/connect_info_256.png</normaloff>:/img/img/connect_info_256.png</iconset>
   </property>
   <property name="text">
    <string>showInfo</string>
   </property>
   <property name="toolTip">
    <string>Показать информацию</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/connect_config_256.png</normaloff>:/img/img/connect_config_256.png</iconset>
   </property>
   <property name="text">
    <string>config</string>
   </property>
   <property name="toolTip">
    <string>Настройка</string>
   </property>
  </action>
  <action name="action_route1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/inverse_kinematics_256_1.png</normaloff>
     <normalon>:/img/img/inverse_kinematics_ok_256_1.png</normalon>:/img/img/inverse_kinematics_256_1.png</iconset>
   </property>
   <property name="text">
    <string>Route1</string>
   </property>
   <property name="toolTip">
    <string>Маршрут 1</string>
   </property>
  </action>
  <action name="action_route2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/inverse_kinematics_256_2.png</normaloff>
     <normalon>:/img/img/inverse_kinematics_ok_256_2.png</normalon>
     <activeon>:/img/img/inverse_kinematics_ok_256_2.png</activeon>:/img/img/inverse_kinematics_256_2.png</iconset>
   </property>
   <property name="text">
    <string>Route2</string>
   </property>
   <property name="toolTip">
    <string>Маршрут 2</string>
   </property>
  </action>
  <action name="action_route3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/inverse_kinematics_256_3.png</normaloff>
     <activeon>:/img/img/inverse_kinematics_ok_256_3.png</activeon>:/img/img/inverse_kinematics_256_3.png</iconset>
   </property>
   <property name="text">
    <string>Route3</string>
   </property>
   <property name="toolTip">
    <string>Маршрут 3</string>
   </property>
  </action>
  <action name="action_add_radio">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/wifi_256.png</normaloff>:/img/img/wifi_256.png</iconset>
   </property>
   <property name="text">
    <string>radio</string>
   </property>
   <property name="toolTip">
    <string>Добавить радиомост</string>
   </property>
  </action>
  <action name="actionRouteInform">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/route/inverse_kinematics_save_256.png</normaloff>
     <normalon>:/img/img/route/inverse_kinematics_save_256.png</normalon>:/img/img/route/inverse_kinematics_save_256.png</iconset>
   </property>
   <property name="text">
    <string>routeInform</string>
   </property>
   <property name="toolTip">
    <string>Сохранить маршрут</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/question.png</normaloff>:/img/img/question.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Сбросить всё</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Начать опрос</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Завершить опрос</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Настройка</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Показать адреса</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Скрыть адреса</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>Показать имена</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>Скрыть имена</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections/>
</ui>
